projects/ui/src/functions/ebipdata/ebipdata.ts

```diff
@@ -0,0 +1,71 @@
+import { Handler } from '@netlify/functions';
+import * as fs from 'fs';
+import middy from 'middy';
+import path from 'path';
+import { cors, rateLimit } from '../middleware';
+import { ebipList } from './ebipList';
+
+/**
+ * Return EBIP content.
+ */
+const _handler: Handler = async (event) => {
+  try {
+
+    if (event.queryStringParameters?.getEbip) {
+      if (event.queryStringParameters?.getEbip === 'all') {
+        return {
+          statusCode: 200,
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(ebipList),
+        };
+      };
+
+      const ebipNumber = Number(event.queryStringParameters?.getEbip);
+      if (ebipNumber >= 0) {
+        const ebipBody = await new Promise((resolve, reject) => {
+          fs.readFile(path.join(__dirname, `./ebips/ebip-${ebipNumber}.md`), 'utf8', (err, data) => {
+            if (err) {
+              return reject(err);
+            }
+            resolve(data);
+          });
+        });
+
+        const output = {
+          ...ebipList[ebipNumber],
+          body: ebipBody,
+        };
+    
+        if (!ebipBody) {
+          return {
+            statusCode: 404,
+          };
+        };
+    
+        return {
+          statusCode: 200,
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify(output),
+        };
+      };
+    };
+
+    return {
+      statusCode: 403
```

Comment:
Should seems like the wrong response code. 403 is for unauthorized. Did you mean 404 not found, or 400 bad request?

Reply:
updated to 404